src/test/java/com/example/solidconnection/application/service/ApplicationSubmissionServiceTest.java

```diff
@@ -38,48 +42,76 @@ class ApplicationSubmissionServiceTest extends BaseIntegrationTest {
     private ApplicationRepository applicationRepository;
 
     @Autowired
-    private GpaScoreRepository gpaScoreRepository;
+    private SiteUserFixture siteUserFixture;
 
     @Autowired
-    private LanguageTestScoreRepository languageTestScoreRepository;
+    private UniversityInfoForApplyFixture universityInfoForApplyFixture;
+
+    @Autowired
+    private LanguageRequirementFixture languageRequirementFixture;
+
+    @Autowired
+    private GpaScoreFixture gpaScoreFixture;
+
+    @Autowired
+    private LanguageTestScoreFixture languageTestScoreFixture;
+
+    @Value("${university.term}")
+    private String term;
+
+    private SiteUser user;
+    private UniversityInfoForApply 괌대학_A_지원_정보;
+    private UniversityInfoForApply 괌대학_B_지원_정보;
+    private UniversityInfoForApply 서던덴마크대학교_지원_정보;
+
+    @BeforeEach
+    void setUp() {
+        user = siteUserFixture.사용자();
+
+        괌대학_A_지원_정보 = universityInfoForApplyFixture.괌대학_A_지원_정보();
+        languageRequirementFixture.토플_80(괌대학_A_지원_정보);
+        languageRequirementFixture.토익_800(괌대학_A_지원_정보);
+
+        괌대학_B_지원_정보 = universityInfoForApplyFixture.괌대학_B_지원_정보();
+        languageRequirementFixture.토플_70(괌대학_B_지원_정보);
+        languageRequirementFixture.토익_900(괌대학_B_지원_정보);
+
+        서던덴마크대학교_지원_정보 = universityInfoForApplyFixture.서던덴마크대학교_지원_정보();
+        languageRequirementFixture.토플_70(서던덴마크대학교_지원_정보);
```

Comment:
```suggestion
        괌대학_A_지원_정보 = universityInfoForApplyFixture.괌대학_A_지원_정보();
        괌대학_B_지원_정보 = universityInfoForApplyFixture.괌대학_B_지원_정보();
        서던덴마크대학교_지원_정보 = universityInfoForApplyFixture.서던덴마크대학교_지원_정보();
```

이부분도 성적 요구사항 없어도 통과합니다~ 
테스트 목적에 맞는 것들만 설정해서 복잡도를 줄어봐요 😁